spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -119,4 +138,10 @@ def stub_publishing_api
     stub_any_publishing_api_put_content
     stub_any_publishing_api_publish
   end
+
+  def create_step_by_step_page
+    build(:step_by_step_page_with_steps).tap do |step_page|
+      step_page.save(validate: false)
+    end
+  end
```

Comment:
Can we avoid creating a new method and instead do something like this on line 34?

```ruby
let(:step_by_step_page) { create(:step_by_step_page_with_steps) }
```

Reply:
You should be able to re-use the `step_by_step_page` created on line 7.

Reply:
yes, I don't need that and I don't need line 35 either!